src/actions/messageActions.ts

```diff
@@ -105,10 +105,9 @@ export const setMainPoint = (
 };
 
 export interface CombinePointsParams {
-  aboveOrBelow: "above" | "below";
-  point: PointI;
   shape: PointShape;
-  index: number;
+  keepIndex: number;
+  deleteIndex: number;
```

Comment:
Very good - as I was working on normalizing state today, I realized that a change like this had to be done.